src/EA.Weee.RequestHandlers/Admin/GetAllAatfsForComplianceYearHandler.cs

```diff
@@ -35,7 +36,9 @@ public async Task<List<OrganisationSchemeData>> HandleAsync(GetAllAatfsForCompli
                 throw new ArgumentException($"Aatfs not found for a compliance year {message.ComplianceYear}");
             }
 
-            return mapper.Map<List<Domain.AatfReturn.Aatf>, List<OrganisationSchemeData>>(listAatfs);
+            var result = mapper.Map<List<Domain.AatfReturn.Aatf>, List<OrganisationSchemeData>>(listAatfs);
```

Comment:
Wonder if we are using OrganisationSchemeData for a more generic Id, DisplayName pair, perhaps we can rename it to something more generic... Maybe EntityIdDisplayNameData or something :)

Reply:
Yes, I agree with that. Also needs to be put in a Shared place. I will do it.